lib/mongoose_push/config/provider/toml.ex

```diff
@@ -17,11 +17,13 @@ defmodule MongoosePush.Config.Provider.Toml do
       current_sysconfig = Application.get_all_env(@app)
       updated_sysconfig = update_sysconfig(current_sysconfig, toml_config)
 
+      config = maybe_erase_confex_services(config, updated_sysconfig)
```

Comment:
I have a question: how do we know that the Confex provider is applied before the TOML one?

Reply:
The order is defined in `mix.exs` [here](https://github.com/esl/MongoosePush/blob/master/mix.exs#L201). Actually it's the other way - TOML is applied before Confex. Here we're erasing Confex entries before they're even resolved. For instance, in [prod.exs](https://github.com/esl/MongoosePush/blob/master/config/prod.exs#L48) we have a `default` fcm pool which is composed of Confex entries in format `{:system, type, var_name, default}`. We want to ensure that this whole pool is erased if other pools are defined by TOML config. Same goes for apns.

Reply:
Sounds good. I knew the order from `mix.exs`, but I incorrectly assumed that we are removing the resolved pools.